jcl/src/java.management/share/classes/java/lang/management/DefaultPlatformMBeanProvider.java

```diff
@@ -61,9 +67,18 @@
 			.addInterface(java.lang.management.OperatingSystemMXBean.class)
 			.register(allComponents);
 
-		ComponentBuilder.create(LoggingMXBeanImpl.getInstance())
-			.addInterface(java.lang.management.PlatformLoggingMXBean.class)
-			.register(allComponents);
+		/* LoggingMXBeanImpl depends on java.logging. If java.logging is not 
+		 * available exclude this component 
+		 */
+/*[IF Sidecar19-SE-B165]*/
+		if (ModuleLayer.boot().findModule("java.logging").isPresent()) { //$NON-NLS-1$
+/*[ELSE]
+		if (Layer.boot().findModule("java.logging").isPresent()) { //$NON-NLS-1$
```

Comment:
I'd like to see the opening `{` on lines 74 and 76 moved outside the preprocessor conditional.